playbooks/common/openshift-cluster/config.yml

```diff
@@ -17,6 +17,24 @@
       - package_version
       - docker_image_availability
       - docker_storage
+  when: ansible_distribution != "Fedora"
+
+- name: Verify Requirements - Fedora
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - docker_image_availability
+      - docker_storage
+  when: ansible_distribution == "Fedora"
```

Comment:
Yeah, I missed the `package_availability` in my [add dnf support patch](https://github.com/openshift/openshift-ansible/pull/5101). I guess I could add that later but if there's not much value in it then I can just leave it. I'll make them a single play.